bundles/org.openhab.binding.homewizard/src/main/java/org/openhab/binding/homewizard/internal/HomeWizardConfiguration.java

```diff
@@ -37,10 +37,14 @@ public class HomeWizardConfiguration {
     /**
      * The API version to be used
      */
-    public Integer apiVersion = 1;
+    public String apiVersion = "2";
 
     /**
      * Bearer token for API v2
      */
     public String bearerToken = "";
+
+    public Boolean useApiVersion2() {
```

Comment:
useApiVersion2() suggests it is a command/setter not a conditional check. i suggest `isUsingApiV2` or similar